cmake/Modules/MacroRootDict.cmake

```diff
@@ -116,21 +115,36 @@ MACRO( GEN_ROOT_DICT_SOURCE _dict_src_filename )
     # need to prefix all include dirs with -I
     set( _dict_includes )
     FOREACH( _inc ${ROOT_DICT_INCLUDE_DIRS} )
-        SET( _dict_includes "${_dict_includes}\t-I${_inc}")  #fg: the \t fixes a wired string expansion 
+        SET( _dict_includes "${_dict_includes}\t-I${_inc}")  #fg: the \t fixes a wired string expansion
```

Comment:
Be consistent with the spaces: `SET(<space>...<space>)`

Reply:
I think this line was just a formatting change (i.e. tab to whitespace), because you see the code itself is the same. OK, added the fix for the space at the end.